hack/test/e2e-qemu.sh

```diff
@@ -262,6 +262,18 @@ case "${WITH_ENFORCING:-false}" in
     ;;
 esac
 
+case "${WITH_AIRGAPPED:-false}" in
+  false)
+    ;;
+  *)
+    "${TALOSCTL}" debug air-gapped --advertised-address 172.20.1.1 >/tmp/airgapped.log 2>&1 &
```

Comment:
we should add this to additional artifacts in kres yanl otherwise won't get uploaded

Reply:
I did

Reply:
yeah, it's saved

Reply:
ahh I was blind